.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,54 @@
+name: Test Dependabot PRs
+
+on:
+  pull_request:
+
+env:
+  GU_SUPPORT_WORKERS_LOAD_S3_CONFIG: false
+
+jobs:
+  run_node_tests:
+    if: github.actor == 'dependabot[bot]'
+    name: test dependabot
+    runs-on: ubuntu-latest
+    steps:
+      # ---- Logging ---- #
+      - name: Env
+        run: env
+
+      - name: Dump GitHub context
+        id: github_context_step
+        run: echo $JSON
+        env:
+          JSON: ${{ toJSON(github) }}
+
+      # ---- Setup ---- #
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: guardian/actions-setup-node@v2.4.1
+        with:
+          cache: 'yarn'
+          cache-dependency-path: yarn.lock
+
+      - name: Install - support-reminders
+        run: yarn
+        working-directory: /
+
+      - name: Install - CDK
+        run: yarn
+        working-directory: cdk
+
+      # ---- Test ---- #
+      - name: Test - support-reminders
+        run: yarn run test
+        working-directory: /
+
+      - name: Test - CDK
+        run: yarn run test
+        working-directory: cdk
+
```

Comment:
This will run the test commands in the `package.json` in the root for `support-reminders` and the `package.json` in the `cdk` directory - which is good. I actually think we should also validate by running the build commands in each of the respective repos, which is `yarn run build` for both. i say this as the `dependabot` PRs may update dependencies used at build time.

Reply:
I have added the  build commands in each of the respective package.json files to the GitHub workflow.I have changed the test related commands as the support reminder unit test cases are to be run on a Postgresql test database